### DISCUSSION TITLE
可以选择只绘制某个省的省界线吗？

### DISCUSSION THREAD

**Discussion by cuizilu** [gmt-china/china-geospatial-data#51] [Q&A] [answered]
感谢你们做出的重要贡献，为大家带来了许多便利！
同时想请问这个数据集调用的时候可以选择只调用某个省的数据吗？或者能否在CN-border-La.gmt里增加注释，把全国各个省的数据区分开，让用户可以自己切分呢？

**Comment by seisman** [comment, marked as answer]
目前的数据还不支持选择单个省界，未来会考虑把这部分信息加进去，详细讨论见 https://github.com/gmt-china/china-geospatial-data/issues/41
